Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.h

```diff
@@ -20,8 +20,12 @@ namespace ROS2
         static void Reflect(AZ::ReflectContext* context);
 
         float m_verticalFieldOfViewDeg = 90.0f; //!< Vertical field of view of camera sensor.
+        float m_minVerticalFieldOfViewDeg = 0.0f;
```

Comment:
These limits are const values and not reflected. Could we group / separate them into private internal structure called Limits?